electron-app/package.json

```diff
@@ -97,22 +97,31 @@
-    "vscode.log": "https://open-vsx.org/api/vscode/log/1.51.1/file/vscode.log-1.51.1.vsix",
-    "vscode.markdown": "https://open-vsx.org/api/vscode/markdown/1.51.1/file/vscode.markdown-1.51.1.vsix",
-    "vscode.typescript": "https://open-vsx.org/api/vscode/typescript/1.51.1/file/vscode.typescript-1.51.1.vsix",
-    "vscode.typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.51.1/file/vscode.typescript-language-features-1.51.1.vsix",
-    "vscode.yaml": "https://open-vsx.org/api/vscode/yaml/1.51.1/file/vscode.yaml-1.51.1.vsix",
-    "vscode.xml": "https://open-vsx.org/api/vscode/xml/1.51.1/file/vscode.xml-1.51.1.vsix"
+    "ms-vscode.js-debug": "https://open-vsx.org/api/ms-vscode/js-debug/1.52.2/file/ms-vscode.js-debug-1.52.2.vsix",
+    "vscode.css": "https://open-vsx.org/api/vscode/css/1.52.1/file/vscode.css-1.52.1.vsix",
+    "vscode.css-language-features": "https://open-vsx.org/api/vscode/css-language-features/1.52.1/file/vscode.css-language-features-1.52.1.vsix",
+    "vscode.docker": "https://open-vsx.org/api/vscode/docker/1.52.1/file/vscode.docker-1.52.1.vsix",
+    "vscode.html": "https://open-vsx.org/api/vscode/html/1.52.1/file/vscode.html-1.52.1.vsix",
+    "vscode.html-language-features": "https://open-vsx.org/api/vscode/html-language-features/1.52.1/file/vscode.html-language-features-1.52.1.vsix",
+    "vscode.image-preview": "https://open-vsx.org/api/vscode/image-preview/1.52.1/file/vscode.image-preview-1.52.1.vsix",
+    "vscode.ini": "https://open-vsx.org/api/vscode/ini/1.52.1/file/vscode.ini-1.52.1.vsix",
+    "vscode.javascript": "https://open-vsx.org/api/vscode/javascript/1.52.1/file/vscode.javascript-1.52.1.vsix",
+    "vscode.json": "https://open-vsx.org/api/vscode/json/1.52.1/file/vscode.json-1.52.1.vsix",
+    "vscode.json-language-features": "https://open-vsx.org/api/vscode/json-language-features/1.52.1/file/vscode.json-language-features-1.52.1.vsix",
+    "vscode.log": "https://open-vsx.org/api/vscode/log/1.52.1/file/vscode.log-1.52.1.vsix",
+    "vscode.markdown": "https://open-vsx.org/api/vscode/markdown/1.52.1/file/vscode.markdown-1.52.1.vsix",
+    "vscode.npm": "https://open-vsx.org/api/vscode/npm/1.52.1/file/vscode.npm-1.52.1.vsix",
+    "vscode.powershell": "https://open-vsx.org/api/vscode/powershell/1.52.1/file/vscode.powershell-1.52.1.vsix",
+    "vscode.shellscript": "https://open-vsx.org/api/vscode/shellscript/1.52.1/file/vscode.shellscript-1.52.1.vsix",
+    "vscode.typescript": "https://open-vsx.org/api/vscode/typescript/1.52.1/file/vscode.typescript-1.52.1.vsix",
+    "vscode.typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.52.1/file/vscode.typescript-language-features-1.52.1.vsix",
+    "vscode.yaml": "https://open-vsx.org/api/vscode/yaml/1.52.1/file/vscode.yaml-1.52.1.vsix",
+    "vscode.xml": "https://open-vsx.org/api/vscode/xml/1.52.1/file/vscode.xml-1.52.1.vsix"
```

Comment:
I think the `node-debug` extension(s) is/are necessary to debug JS/TS, e.g. if we want to use theapp to debug during Theia development?

e.g.: https://github.com/eclipse-theia/theia/blob/master/package.json#L121-L122

Reply:
Thanks, we'll add those as well.

Reply:
Added here: https://github.com/eclipse-theia/theia-example/pull/24/commits/0d8df2022a07aedd3b6f694d0ead3df7b5e290a6